images/linux/scripts/installers/postgresql.sh

```diff
@@ -7,10 +7,22 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
+#Preparing repo for PostgreSQL 12.
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+
+echo "Install PostgreSQL"
+apt update
+apt install postgresql postgresql-client
+
 echo "Install libpq-dev"
 apt-get install libpq-dev
 
-echo "Install Postgresql Client"
-apt-get install postgresql-client
+#Verify that PostgreSQL is ready for accept incoming connections.
+# exit codes:
+# ready - 0
+# reject - 1
+# connection timeout - 2
+pg_isready
```

Comment:
Will this fail script if PostgreSQL is not available

Reply:
As far as I know, In case of any exit code except zero, will automatically fail the script. I have checked on MacOS, Ubuntu images, that exit code depends on postgresql status. Is it enough to accept this part of code?

Reply:
If it fails, it is okay for me. Thanks!

Just an option, we can add `set -e` at the begin of script